src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/DungeonFeatures.kt

```diff
@@ -76,10 +78,15 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 import net.minecraftforge.fml.common.gameevent.TickEvent
 import net.minecraftforge.fml.common.gameevent.TickEvent.ClientTickEvent
 import java.awt.Color
+import kotlin.random.Random
 
 object DungeonFeatures {
     private val deathOrPuzzleFail =
         Regex("^ ☠ .+ and became a ghost\\.$|^PUZZLE FAIL! .+$|^\\[STATUE] Oruo the Omniscient: .+ chose the wrong answer!")
+    private val jaxForgedRegex = Regex("§r§aJax forged §r§.+§r§8 x(?<amount>\\d+)§r§a.*!§r")
```

Comment:
I could probably use this to re-estimate the arrow count but idk how reliable it would be